vulcanai/tests/datasets/test_tabulardataset.py

```diff
@@ -1,144 +1,144 @@
-# coding=utf-8
-""" Defines test cases for tabular dataset """
-import pytest
```

Comment:
woah that's a lot of deletions. Are you working to adapt these tests for the new a-la-carte function structure?

Reply:
haha yes, I wrote that in the commit message but should have said it here. I'll add them back once we've decided on the final structure... would rather not spend time writing tests I don't have to

Reply:
got it - sorry I missed that